lib/actor-system.js

```diff
@@ -769,8 +769,12 @@ class ActorSystem {
     if (config.clusterSize > 1) {
       let balancerActor = await this._createBalancerActor(definition, parent, config);
 
-      let childPromises = _.times(config.clusterSize, () =>
-        balancerActor.createChild(definition, _.extend({}, config, { clusterSize: 1 })));
+      let childPromises = _.times(config.clusterSize, n =>
```

Comment:
I don't see why we need to set actor number to `"threaded"` actors: they don't spawn a sub-process, so we don't have to deal with command line arguments in this case at all. I guess, instead we need to do this change in `_createForkedActor` method. Your PR is about forked actors, isn't it?

Reply:
Thank you for the review. I accidentally pasted the changes into the wrong context.